plugins/HandwritingRecognition/plugin.ini

```diff
@@ -0,0 +1,15 @@
+[about]
+## Author / Copyright notice
+author=Roland Lötscher
+
+description=Issue https://github.com/xournalpp/xournalpp/issues/2152
+
+## If the plugin is packed with Xournal++, use
+## <xournalpp> then it gets the same version number
+version=<xournalpp>
```

Comment:
Hello! I don't understand. We should replace <xournallpp> with the actual version number?